### PR TITLE
Unlogged layout

### PR DESCRIPTION
**Pull request by nabucopena**
We create a new layout to show when the user isn't logged, it doesn't show in the navbar the routes that can only be used by a logged user.

### CODE REVIEW

File: migrations/4_create_messages.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+create table messages (
```
Comment (carlosipe) [inline migrations/4_create_messages.sql:1]: This seems like it doesn't belong to this PR.

You can remove that commit by running `git rebase -i HEAD~2` and then using the `drop` command instead of `pick` in the commit you want to drop.

File: views/homepage.mote [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-	<p>Log in with your account</p>
-	<form method="post" action="/login">
-		<input type="text" name="username"/>
-		<input type="password" name="password"/>
-		<button type="submit" class="btn btn-primary">login</button>
-	</form>
-	<form method="get" action="/register">
-		<button type="submit" class="btn btn-primary">Register</button>
-	</form>
+	% if !this.session[:user]
```
Comment (carlosipe) [inline views/homepage.mote:3]: What do you think about using the `current_user` helper here instead?

Also, when negating conditions in Ruby sometimes it's more idiomatic to use `unless`. Let's say something like

```rb
unless current_user
```

File: app.rb [hunk 3]
```diff
@@ -187,7 +187,11 @@ def conn
       render("show_results", results: results)
     end
     on root do
-      render "homepage"
+      if session[:user]
+        render "homepage"
+      else
+        render("homepage", {}, "unlogged_layout")
```
Comment (carlosipe) [inline app.rb:193]: Just curious, what do we need the "homepage" page for? Can't we just redirect to "/login" if the user is not logged in and to "/games" if logged?

Comment (nabucopena) [reply app.rb:193]: actually, this page isn't needed, i just thought make it, maybe could delete the /login page and redirect to the homepage

File: migrations/4_create_messages.sql (hunk 1 above)
Comment (nabucopena) [reply migrations/4_create_messages.sql]: done

File: views/homepage.mote (hunk 2 above)
Comment (nabucopena) [reply views/homepage.mote]: done
